### DISCUSSION TITLE
Helm upgrade with spesific appversion

### DISCUSSION THREAD

**Discussion by realhuseyn** [linkerd/linkerd2#12370] [Help]
Hello. I always upgrade with the following command:
helm upgrade --install linkerd-control-plane -n linkerd  linkerd/linkerd-control-plane -f values-linkerd-controlplane.yaml
But when I execute that command, it upgrades the latest version. I want to upgrade to the specific linkerd app version. I inspect that there is an appVersion field in chart.yaml. But I don't know how can I use it in helm upgrade command.

**Comment by alpeb** [comment]
I believe you can use helm's `--version` flag?
